addons/l10n_uy/models/template_uy.py

```diff
@@ -24,7 +27,6 @@ def _get_uy_res_company(self):
                 'bank_account_code_prefix': '1111',
                 'cash_account_code_prefix': '1112',
                 'transfer_account_code_prefix': '11120',
-                'account_default_pos_receivable_account_id': 'uy_code_11307',
```

Comment:
we remove it because the account does not exist aymore.
If you think we need to define a default one please @MarianaLuzTorres @fvz-odoo  @ren-odoo could you help us with this?
Best Regards

Reply:
@zaoral as agreed with @marianaluzTorres let's add the account back :)

thank you.

Reply:
Thank you all.
I have re-added the account.
Best Regards
@ren-odooh @MarianaLuzTorres @jco-odoo